packages/amplify-category-function/package.json

```diff
@@ -35,7 +35,7 @@
     "folder-hash": "^4.0.2",
     "fs-extra": "^8.1.0",
     "globby": "^11.0.3",
-    "graphql-transformer-core": "^7.6.9",
+    "graphql-transformer-core": "7.6.10-upgrade-graphql15-2.1",
```

Comment:
These will all be changed to the production release with GraphQL 15 when it is available.